solidity/contracts/middleware/InterchainAccountRouter.sol

```diff
@@ -1138,11 +1077,9 @@ contract InterchainAccountRouter is Router, AbstractRoutingIsm {
         return
             _Router_quoteDispatch(
                 _destination,
-                bytes(""),
-                StandardHookMetadata.overrideGasLimit(
-                    gasLimit + COMMIT_TX_GAS_USAGE
-                ),
+                new bytes(0),
+                StandardHookMetadata.overrideGasLimit(COMMIT_TX_GAS_USAGE),
                 address(hook)
-            );
+            ) + quoteGasPayment(_destination, gasLimit);
```

Comment:
Do we need two calls to `Router_quoteDispatch` if we're just passing gas limits to the IGP?

Reply:
discussed offline, there can be other variable and fixed costs which do not scale linearly with the gas limit